### PR TITLE
Bootstrap styles

### PR DESCRIPTION
**Pull request by meef3k**
- added views with bootstrap for seances/index and movies/id
- added navbar
- fixed seances method which calculate finishes_at

### CODE REVIEW

File: app/controllers/seances_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class SeancesController < ApplicationController
   # GET /seances
   def index
     @seances = Seance.includes(:movie, :hall).all
+    # movie.seances.all
```
Comment (Protoplaste) [inline app/controllers/seances_controller.rb]: This propably doesn't need to be here

File: app/models/seance.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Seance < ApplicationRecord
   belongs_to :hall
   belongs_to :movie
-  before_validation :set_finishes_at
+  before_validation :start_nil?
```
Comment (Protoplaste) [inline app/models/seance.rb]: I sugest avoiding making a method that just calls another method if a condition is met, it's needlesly hard to follow, instead we can extract the condition out like this:
```suggestion
  before_validation :set_finishes_at, if: :start_nil?
```
And of course we need to modify the `start_nil?` method so that it actually tells us if `start_at` is nill as currently it doesn't do what it says it does. 
If a method name is ending in ? it is generally expected to return true or false

**Review by Protoplaste — CHANGES_REQUESTED** [review]

File: app/models/seance.rb (hunk 2 above)
Comment (meef3k) [reply app/models/seance.rb]: I've changed what you pointed out. Thanks :D

**Comment by meef3k** [comment]
- Changed seances validation
- Edited views for seances and movies
- Edited controllers for seances and movies

File: config/initializers/constants.rb [hunk 3]
```diff
@@ -0,0 +1 @@
+ADS_TIME = 60.minutes
```
Comment (Protoplaste) [inline config/initializers/constants.rb:1]: Are we sure ad times will always be constant?
Is there no possibility for different seances to have different ad times?

Comment (meef3k) [reply config/initializers/constants.rb:1]: Tobiasz told us to set it as constant, for example 30 minutes. But I also added time for cleaning the hall.

Comment (Protoplaste) [reply config/initializers/constants.rb:1]: I see, ok, no problem then :)

File: app/controllers/seances_controller.rb [hunk 4]
```diff
@@ -1,7 +1,10 @@
 class SeancesController < ApplicationController
   # GET /seances
   def index
-    @seances = Seance.includes(:movie, :hall).all
+    @seances = Seance
+               .includes(:movie)
+               .where(starts_at: DateTime.current..Date.today + 7.days)
```
Comment (Protoplaste) [inline app/controllers/seances_controller.rb]: This might be simplified to:
```suggestion
               .where(starts_at: DateTime.current..7.days.from_now)
```

File: app/controllers/movies_controller.rb [hunk 5]
```diff
@@ -1,5 +1,9 @@
 class MoviesController < ApplicationController
   def show
     @movie = Movie.find(params[:id])
+    @movieseances = Seance
+                    .where(movie_id: params[:id])
+                    .where(starts_at: DateTime.current..Date.today + 7.days)
```
Comment (Protoplaste) [inline app/controllers/movies_controller.rb]: This might be simplified to:
```suggestion
                    .where(starts_at: DateTime.current..7.days.from_now)
```

**Review by Protoplaste — APPROVED** [review]
Aside from some potential simplifications LGTM
